src/validators/alphaNumeric.js

```diff
@@ -0,0 +1,15 @@
+import i18n from '@dhis2/d2-i18n'
+import { isEmpty, isString } from './helpers/index.js'
+
+const ALPHA_NUMERIC_PATTERN = /^[a-z0-9]*$/i
```

Comment:
I think this should allow spaces. I'd expect "This is a text" to be a valid alpha numeric value. If you agree, please add a test value to the test

Reply:
OK, agreed and fixed in https://github.com/dhis2/ui-forms/pull/164/commits/98990f43d6258af21b8184e8bf206a30b8ef8d06